components/setupGame/MatchList.tsx

```
@@ -289,27 +479,105 @@ const MatchList: FC<MatchListProps> = ({
+    const animations = Object.values(fadeAnims.current).map((anim) =>
+      Animated.timing(anim, {
+        toValue: 0,
+        duration: 300,
+        useNativeDriver: true,
+      })
+    );
+
+    if (animations.length === 0) {
+      // If no animations exist, clear immediately
+      setGlobalMatches && setGlobalMatches([]);
+      return;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)